2021/02/15 04:32:44 : products_errors 9: Traceback (most recent call last):
  File "/var/www/python/cartmigration/controllers/migration.py", line 1036, in migration
    convert = fn_convert_export(main, ext)
  File "/var/www/python/cartmigration/models/cart/custom.py", line 540, in convert_product_export
    product_data['name'] = html.unescape(products_ext_data['products_name'])
KeyError: 'products_name'

2021/02/15 04:32:45 : products_errors 10: Traceback (most recent call last):
  File "/var/www/python/cartmigration/controllers/migration.py", line 1036, in migration
    convert = fn_convert_export(main, ext)
  File "/var/www/python/cartmigration/models/cart/custom.py", line 540, in convert_product_export
    product_data['name'] = html.unescape(products_ext_data['products_name'])
KeyError: 'products_name'

2021/02/15 04:32:46 : products_errors 16: Traceback (most recent call last):
  File "/var/www/python/cartmigration/controllers/migration.py", line 1036, in migration
    convert = fn_convert_export(main, ext)
  File "/var/www/python/cartmigration/models/cart/custom.py", line 540, in convert_product_export
    product_data['name'] = html.unescape(products_ext_data['products_name'])
KeyError: 'products_name'

2021/02/15 04:32:46 : products_errors 17: Traceback (most recent call last):
  File "/var/www/python/cartmigration/controllers/migration.py", line 1036, in migration
    convert = fn_convert_export(main, ext)
  File "/var/www/python/cartmigration/models/cart/custom.py", line 540, in convert_product_export
    product_data['name'] = html.unescape(products_ext_data['products_name'])
KeyError: 'products_name'

2021/02/15 04:32:47 : products_errors 18: Traceback (most recent call last):
  File "/var/www/python/cartmigration/controllers/migration.py", line 1036, in migration
    convert = fn_convert_export(main, ext)
  File "/var/www/python/cartmigration/models/cart/custom.py", line 540, in convert_product_export
    product_data['name'] = html.unescape(products_ext_data['products_name'])
KeyError: 'products_name'

2021/02/15 04:32:47 : products_errors 19: Traceback (most recent call last):
  File "/var/www/python/cartmigration/controllers/migration.py", line 1036, in migration
    convert = fn_convert_export(main, ext)
  File "/var/www/python/cartmigration/models/cart/custom.py", line 540, in convert_product_export
    product_data['name'] = html.unescape(products_ext_data['products_name'])
KeyError: 'products_name'

2021/02/15 04:32:48 : products_errors 20: Traceback (most recent call last):
  File "/var/www/python/cartmigration/controllers/migration.py", line 1036, in migration
    convert = fn_convert_export(main, ext)
  File "/var/www/python/cartmigration/models/cart/custom.py", line 540, in convert_product_export
    product_data['name'] = html.unescape(products_ext_data['products_name'])
KeyError: 'products_name'

2021/02/15 04:32:48 : products_errors 22: Traceback (most recent call last):
  File "/var/www/python/cartmigration/controllers/migration.py", line 1036, in migration
    convert = fn_convert_export(main, ext)
  File "/var/www/python/cartmigration/models/cart/custom.py", line 540, in convert_product_export
    product_data['name'] = html.unescape(products_ext_data['products_name'])
KeyError: 'products_name'

2021/02/15 04:32:48 : products_errors 23: Traceback (most recent call last):
  File "/var/www/python/cartmigration/controllers/migration.py", line 1036, in migration
    convert = fn_convert_export(main, ext)
  File "/var/www/python/cartmigration/models/cart/custom.py", line 540, in convert_product_export
    product_data['name'] = html.unescape(products_ext_data['products_name'])
KeyError: 'products_name'

2021/02/15 04:32:49 : products_errors 24: Traceback (most recent call last):
  File "/var/www/python/cartmigration/controllers/migration.py", line 1036, in migration
    convert = fn_convert_export(main, ext)
  File "/var/www/python/cartmigration/models/cart/custom.py", line 540, in convert_product_export
    product_data['name'] = html.unescape(products_ext_data['products_name'])
KeyError: 'products_name'

2021/02/15 04:32:49 : products_errors 25: Traceback (most recent call last):
  File "/var/www/python/cartmigration/controllers/migration.py", line 1036, in migration
    convert = fn_convert_export(main, ext)
  File "/var/www/python/cartmigration/models/cart/custom.py", line 540, in convert_product_export
    product_data['name'] = html.unescape(products_ext_data['products_name'])
KeyError: 'products_name'

2021/02/15 04:32:50 : products_errors 37: Traceback (most recent call last):
  File "/var/www/python/cartmigration/controllers/migration.py", line 1036, in migration
    convert = fn_convert_export(main, ext)
  File "/var/www/python/cartmigration/models/cart/custom.py", line 540, in convert_product_export
    product_data['name'] = html.unescape(products_ext_data['products_name'])
KeyError: 'products_name'

2021/02/15 04:32:50 : products_errors 39: Traceback (most recent call last):
  File "/var/www/python/cartmigration/controllers/migration.py", line 1036, in migration
    convert = fn_convert_export(main, ext)
  File "/var/www/python/cartmigration/models/cart/custom.py", line 540, in convert_product_export
    product_data['name'] = html.unescape(products_ext_data['products_name'])
KeyError: 'products_name'

2021/02/15 04:32:50 : products_errors 41: Traceback (most recent call last):
  File "/var/www/python/cartmigration/controllers/migration.py", line 1036, in migration
    convert = fn_convert_export(main, ext)
  File "/var/www/python/cartmigration/models/cart/custom.py", line 540, in convert_product_export
    product_data['name'] = html.unescape(products_ext_data['products_name'])
KeyError: 'products_name'

2021/02/15 04:32:51 : products_errors 42: Traceback (most recent call last):
  File "/var/www/python/cartmigration/controllers/migration.py", line 1036, in migration
    convert = fn_convert_export(main, ext)
  File "/var/www/python/cartmigration/models/cart/custom.py", line 540, in convert_product_export
    product_data['name'] = html.unescape(products_ext_data['products_name'])
KeyError: 'products_name'

2021/02/15 04:32:51 : products_errors 43: Traceback (most recent call last):
  File "/var/www/python/cartmigration/controllers/migration.py", line 1036, in migration
    convert = fn_convert_export(main, ext)
  File "/var/www/python/cartmigration/models/cart/custom.py", line 540, in convert_product_export
    product_data['name'] = html.unescape(products_ext_data['products_name'])
KeyError: 'products_name'

2021/02/15 04:32:51 : products_errors 44: Traceback (most recent call last):
  File "/var/www/python/cartmigration/controllers/migration.py", line 1036, in migration
    convert = fn_convert_export(main, ext)
  File "/var/www/python/cartmigration/models/cart/custom.py", line 540, in convert_product_export
    product_data['name'] = html.unescape(products_ext_data['products_name'])
KeyError: 'products_name'

2021/02/15 04:32:52 : products_errors 47: Traceback (most recent call last):
  File "/var/www/python/cartmigration/controllers/migration.py", line 1036, in migration
    convert = fn_convert_export(main, ext)
  File "/var/www/python/cartmigration/models/cart/custom.py", line 540, in convert_product_export
    product_data['name'] = html.unescape(products_ext_data['products_name'])
KeyError: 'products_name'

2021/02/15 04:32:52 : products_errors 48: Traceback (most recent call last):
  File "/var/www/python/cartmigration/controllers/migration.py", line 1036, in migration
    convert = fn_convert_export(main, ext)
  File "/var/www/python/cartmigration/models/cart/custom.py", line 540, in convert_product_export
    product_data['name'] = html.unescape(products_ext_data['products_name'])
KeyError: 'products_name'

2021/02/15 04:32:53 : products_errors 49: Traceback (most recent call last):
  File "/var/www/python/cartmigration/controllers/migration.py", line 1036, in migration
    convert = fn_convert_export(main, ext)
  File "/var/www/python/cartmigration/models/cart/custom.py", line 540, in convert_product_export
    product_data['name'] = html.unescape(products_ext_data['products_name'])
KeyError: 'products_name'

2021/02/15 04:32:53 : customers_errors 1: Traceback (most recent call last):
  File "/var/www/python/cartmigration/controllers/migration.py", line 1036, in migration
    convert = fn_convert_export(main, ext)
  File "/var/www/python/cartmigration/models/cart/custom.py", line 905, in convert_customer_export
    customer_data['id'] = customer['id']
KeyError: 'id'

2021/02/15 04:32:53 : customers_errors 3: Traceback (most recent call last):
  File "/var/www/python/cartmigration/controllers/migration.py", line 1036, in migration
    convert = fn_convert_export(main, ext)
  File "/var/www/python/cartmigration/models/cart/custom.py", line 905, in convert_customer_export
    customer_data['id'] = customer['id']
KeyError: 'id'

